tests/actions.go

```diff
@@ -1329,15 +1329,15 @@ func cloneOperatorRepo() error {
 	return nil
 }
 
-func checkoutTag(tagName string) error {
-	cmd := fmt.Sprintf(`cd /tidb-operator &&
+func (oa *operatorActions) checkoutTag(tagName string) error {
+	cmd := fmt.Sprintf(`cd %s &&
 		git stash -u &&
 		git checkout %s &&
 		mkdir -p /charts/%s &&
 		cp -rf charts/tidb-operator /charts/%s/tidb-operator &&
 		cp -rf charts/tidb-cluster /charts/%s/tidb-cluster &&
 		cp -rf charts/tidb-backup /charts/%s/tidb-backup`,
```

Comment:
(do not block this PR) Is it possible to copy them to temporary directories which are always writable in the future? I guess the path does not matter here. Then we can simplify the workflow (do not need to prepare `/charts`).

Reply:
Agreed for future work, I've tried but found that `/charts` in hard coded in many places😅